### PR TITLE
add example for docker to the cookbook

### PR DESCRIPTION
**Pull request by fmoessle**

### CODE REVIEW

File: cookbook/docker/example.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+## Add a Dockerfile
+```sh
+$ touch Dockerfile
+```
+
+### Dockerfile
+```
+FROM node:11-alpine
```
Comment (bertho-zero) [inline cookbook/docker/example.md]: Prefer an LTS or current version.

Comment (fmoessle) [reply cookbook/docker/example.md]: are you ok with `node:alpine` or `node`?

Comment (bertho-zero) [reply cookbook/docker/example.md]: `node:lts-alpine` ?

**Comment by daffl** [comment]
Wicked. I changed the heading to `Deployment` and added a link to the official NodeJS guide.

**Pull request merged**